cairo/ethereum/cancun/trie.cairo

```diff
@@ -390,6 +392,73 @@ func trie_get_TrieBytes32U256{poseidon_ptr: PoseidonBuiltin*, trie: TrieBytes32U
     return res;
 }
 
+func trie_set_TrieAddressAccount{poseidon_ptr: PoseidonBuiltin*, trie: TrieAddressAccount}(
+    key: Address, value: Account
+) {
+    let dict_ptr_start = cast(trie.value._data.value.dict_ptr_start, DictAccess*);
+    let dict_ptr = cast(trie.value._data.value.dict_ptr, DictAccess*);
+
+    let default = cast(trie.value.default.value, felt);
+    let is_default = Account__eq__(value, trie.value.default);
+
+    with dict_ptr_start, dict_ptr {
+        let (keys) = alloc();
+        assert [keys] = key.value;
+
+        if (is_default.value != 0) {
```

Comment:
to keep consistency, should we cast this ptr to a felt?

Reply:
this is a bool